app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +189,34 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    const documentPublicationActivity = await this.args.meeting.internalDocumentPublicationActivity;
+
+    let hasDocumentPublicationDate = isPresent(documentPublicationActivity.plannedDate);
```

Comment:
I think you need the themis publication activity.
That is what we use in the newsletter service: 
https://github.com/kanselarij-vlaanderen/newsletter-service/blob/master/util/query-helper.js#L212